components/Typography/typography.html

```diff
@@ -0,0 +1,228 @@
+                        <li>
+                            <a href="/Components/Input/input.html">
+                                Input
+                            </a>
+                        </li>
+                        <li>
+                            <a href="/components/Navigation/navigation.html">
+                                Navigation
+                            </a>
+                        </li>
+                        <li>
+                            <a href="/components/Rating/rating.html">
+                                Rating
+                            </a>
+                        </li>
+
+                        <li>
+                            <a href="/components/Toast/toast.html">
+                                Toast
+                            </a>
+                        </li>
+                        <li>
+                            <a href="/components/Modal/modal.html">
+                                Modal
+                            </a>
+                        </li>
+
+                        <li>
+                            <a href="/components/Typography/typography.html">
+                                Typography
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+
+            <div class="doc-content ">
+
+                <section class="Typography">
+                    <h3 class="mb-3 text-primary-color">Typography</h3>
+                    <p class="mb-5 lh-10">Typographies are utility classes used for designing fonts and text.</p>
+                    <h5 class="mt-5 mb-3 text-primary-color">Heading Utilities</h5>
+                    <p class="mb-3 lh-10">For heading text you can use h1, h2, h3, h4, h5, h6 elements.</p>
+
+                    <div class="component-display flex-col">
+                        <h1> H1 heading</h1>
+                        <h2> H2 heading</h2>
+                        <h3> H3 heading</h3>
+                        <h4> H4 heading</h4>
+                        <h5> H5 heading</h5>
+                        <h6> H6 heading</h6>
+                    </div>
+
+                    <div class="code-snippet">
+                        <code>
+                            <iframe
+                                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Ch1%253E%2520H1%2520heading%253C%252Fh1%253E%250A%253Ch2%253E%2520H2%2520heading%253C%252Fh2%253E%250A%253Ch3%253E%2520H3%2520heading%253C%252Fh3%253E%250A%253Ch4%253E%2520H4%2520heading%253C%252Fh4%253E%250A%253Ch5%253E%2520H5%2520heading%253C%252Fh5%253E%250A%253Ch6%253E%2520H6%2520heading%253C%252Fh6%253E"
+                                style="width: 100%; height: auto; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
You can create a class for this instead of using inline style.

Reply:
Yes i have to refactor this in every component.